lms/static/sass/_developer.scss

```diff
@@ -272,4 +272,42 @@
       }
     }
   }
-}
\ No newline at end of file
+}
+
+//efischer part 2: removing team members
+.edit-members {
+  @extend %ui-no-list;
+
+  .team-member {
+    line-height: $body-line-height;
+    padding: 10px;
+  }
+
+  .member-info-container {
+    display: inline-block;
+    vertical-align: middle;
+    margin-left: 10px;
+
+    .primary {
+      font-size: 120%;
+    }
+
+    .secondary {
+      color: $lighter-base-font-color;
+      font-size: 80%;
+      display: block;
+    }
+  }
+
+  .member-profile {
+    img {
+      border: 1px solid $gray;
+    }
+  }
+
+  .action-remove-member {
+    margin-left: 20px;
+    color: $blue;
+  }
+}
+
```

Comment:
TODO: remove all this `_developer.scss` styling once proper UX styles are made and applied in the relevant files.